scipy/stats/stats.py

```diff
@@ -4118,8 +4127,9 @@ def spearmanr(a, b=None, axis=0, nan_policy='propagate'):
         length equal to total number of variables (columns or rows) in ``a``
         and ``b`` combined.
     pvalue : float
-        The two-sided p-value for a hypothesis test whose null hypothesis is
-        that two sets of data are uncorrelated, has same dimension as rho.
+        The p-value for a hypothesis test whose alternative hypothesis is
+        defined by the ``alternative`` parameter and who's null hypotheisis
+        is that two sets of data are uncorrelated, has same dimension as rho.
```

Comment:
What does ", has the same dimension as rho?" mean?

Reply:
That part was left over from the original docstring which I tried to build around. I wasn't sure what to do with it. 
```        
The two-sided p-value for a hypothesis test whose null hypothesis is
that two sets of data are uncorrelated, has same dimension as rho.
```

Reply:
I think this means that the return value `pvalue` has the same shape as the returned `correlation`.  The clause should probably be made a separate sentence.